contract-map.json

```diff
@@ -1,4 +1,11 @@
 {
+  "0x9d29f93e0A4C0bC5AC1E13d5B72038F35C81F325": {
+    "name": "ForgivenetToken",
+    "logo": "frgvn-symbol.png",
```

Comment:
SVG is required

Reply:
DONE :)

Reply:
Hi @MRabenda we are now listed on Alchemy, maybe that works? 
https://dapps.alchemy.com/dapps/1frgvn

Reply:
Hey @kailash-manasarovar - community activity is still low, but I hope with Alechemy listing this will improve and soon I would be able to merge